enhancements/kubelet/user-namespaces-support.md

```diff
@@ -0,0 +1,483 @@
+Thus, `UserNamespacesPodSecurityStandards` is currently in alpha, and will not advance to GA until three releases after Kubelet supports the `UserNamespacesSupport`
+feature gate.
+This feature gate is also the main feature gate used to toggle the SCC changes above. This is done because the same version skew issue exists in Openshift,
+and it would be better to keep UserNamespacesSupport on by default (not skewing from upstrem).
+A consequence of this is the clusterpolicycontroller will also need to be taught to be aware of the `UserNamespacesPodSecurtiyStandards` feature,
+in order to consider these relaxations when assigning a PSA label based on the assigned SCC (if the relaxations are applicable).
+
+`ProcMountType` is not directly related to user namespaces, but it does rely on them, and is useful for podman in Openshift pod use cases. Specifically,
+it's needed to allow podman to configure networking for the sub containers (as mounting a sub '/proc' requires the whole '/proc' be read-write, and '/proc' needs
+to be writable to configure sysctls)
+
+The goal is to graduate this feature in 4.18, meaning `UserNamespacesPodSecurityStandards` and `ProcMountType` will be enabled by default, and will not mark
+a cluster as TechPreviewNoUpgrade.
+
+#### MinimalKubeletVersion in the apiserver
+
+An additional piece is needed for ensuring every node that is in a cluster with `UserNamespacesSupport` enabled actually runs those pods with user namespaces.
+Unfortunately, we cannot rely on homogenous feature gates protecting us in this case. For instance, an admin may pause a worker pool while an upgrade happens, but those workers
+can still be scheduled to. The MCO should not be enabling alpha feature gates for older kubelets, even if the `featuregate` Openshift object enables them [1].
+
+Since there can be a version skew between the kubelet and the kube-apiserver, and the feature gates are not assumed to be homogeneous, then it cannot be assumed that every
+kubelet in the cluster will be new enough to support UserNamespaces.
+
+While this would not be an issue if the featuregates are enabled on kubelet nodes after 4.15, we cannot assume the feature gate will be enabled.
+
+Below is a table that describes some of the version skews between API server and kubelets:
+
+|              | 4.14 api         | 4.15 api         | 4.16 api              | 4.17 api              | 4.18 api              |
+|--------------|------------------|------------------|-----------------------|-----------------------|-----------------------|
+| 4.14 kubelet | ok-no relaxation | ok-no relaxation | bad                   | bad                   | N/A                   |
+| 4.15 kubelet | ok-no relaxation | ok-no relaxation | ok-no idmap in kernel | ok-no idmap in kernel | ok-no idmap in kernel |
+| 4.16 kubelet | ok-no relaxation | ok-no relaxation | ok                    | ok-no idmap in kernel | ok-no idmap in kernel |
```

Comment:
this row, 4.18 API.  If the 4.16 kubelet has the featuregate off (the default), does the field get cleared and the pod created with HostUsers?

Reply:
no this table is only if the featuregate is on. If the featuregate is off, every kubelet ignores ( I mentioned that [here](https://github.com/openshift/enhancements/pull/1649/files#diff-54bd991f708ce4527b3637d1d6628dbdc14d553790c61ea25d0ed23d411f70abR315)